README.md

```diff
@@ -0,0 +1,25 @@
+# Onboarding library
+
+## How to run on Android
+
+1. Install Node.js
+2. Install dependencies from the root of the project
+   ```bash
+   npm install
```

Comment:
Don't you need to also install the dependencies in the example project ?

Reply:
Indeed it was missing